### PR TITLE
fix: add analysis function `print_evm_circuit_row_usage` and `print_rw_usage` to witness block

### PR DESCRIPTION
**Pull request by silathdiir**
Related issue https://github.com/scroll-tech/zkevm-circuits/issues/897 and https://github.com/scroll-tech/zkevm-circuits/issues/906

### CODE REVIEW

File: zkevm-circuits/src/witness/block.rs [hunk 1]
```diff
@@ -130,6 +135,108 @@ impl<F: Field> Block<F> {
     pub(crate) fn get_big_modexp(&self) -> Vec<BigModExp> {
         self.precompile_events.get_modexp_events()
     }
+
+    pub(crate) fn get_num_rows_required_no_padding(&self) -> usize {
```
Comment (lispc) [inline zkevm-circuits/src/witness/block.rs]: rename to print_evm_circuit_row_usage

File: zkevm-circuits/src/witness/block.rs [hunk 2]
```diff
@@ -130,6 +135,108 @@ impl<F: Field> Block<F> {
...
+            log::debug!(
+                "evm circuit row usage: {:?}, step ratio {}, row ratio {}, height {}",
+                e,
+                (usage / height) as f64 / step_num as f64,
+                *usage as f64 / num_rows as f64,
+                height
+            );
+        }
+
+        // EndBlock
```
Comment (lispc) [inline zkevm-circuits/src/witness/block.rs]: remove line170-line175

File: zkevm-circuits/src/witness/block.rs (hunk 1 above)
Comment (silathdiir) [reply zkevm-circuits/src/witness/block.rs]: Renamed in commit [4aa4e81](https://github.com/scroll-tech/zkevm-circuits/pull/933/commits/4aa4e81892a065ce170436fb2d32115f70227570).

File: zkevm-circuits/src/witness/block.rs (hunk 2 above)
Comment (silathdiir) [reply zkevm-circuits/src/witness/block.rs]: Also removed in commit [4aa4e81](https://github.com/scroll-tech/zkevm-circuits/pull/933/commits/4aa4e81892a065ce170436fb2d32115f70227570).

**Review by lispc — APPROVED** [review]

**Pull request merged**